deposit-app/src/test/java/edu/unc/lib/boxc/deposit/fcrepo4/IngestContentObjectsJobTest.java

```diff
@@ -322,29 +331,37 @@ public void ingestWorkObjectTest() throws Exception {
     }
 
     /**
-     * Test that the deposit fails if a file object is specified but not given a
-     * staging location
+     * Test that ingest will work if a file object is specified with no original resource,
+     * assuming it has streaming properties and passed validation
      */
     @Test
-    public void ingestWorkWithFileWithoutLocation() throws Exception {
-        Assertions.assertThrows(JobFailedException.class, () -> {
-            PID workPid = makePid(RepositoryPathConstants.CONTENT_BASE);
-            WorkObject work = mock(WorkObject.class);
-            Bag workBag = setupWork(workPid, work);
+    public void ingestWorkWithFileWithoutOriginalFileAndStagingLocation() {
```

Comment:
The name of the test should probably reflect that the file object is being given streaming properties